src/SFA.DAS.AssessorService.Data/SFA.DAS.AssessorService.Data.csproj

```diff
@@ -17,7 +17,7 @@
 
   <ItemGroup>
     <DotNetCliToolReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Tools" Version="2.0.2" />
-    <PackageReference Include="Dapper" Version="1.50.5" />
+    <PackageReference Include="Dapper" Version="2.0.4" />
```

Comment:
Is this really necessary? - a major version change - check if it's really necessary first....